Dockerfile-oss

```diff
@@ -0,0 +1,115 @@
+FROM --platform=$BUILDPLATFORM maven:3.6.3-jdk-8-slim as builder
+
+WORKDIR /build
+
+COPY pom.xml ./
+COPY management-api-agent/pom.xml ./management-api-agent/pom.xml
+COPY management-api-common/pom.xml ./management-api-common/pom.xml
+COPY management-api-server/pom.xml ./management-api-server/pom.xml
+COPY management-api-shim-3.x/pom.xml ./management-api-shim-3.x/pom.xml
+COPY management-api-shim-4.x/pom.xml ./management-api-shim-4.x/pom.xml
+# this duplicates work done in the next steps, but this should provide
+# a solid cache layer that only gets reset on pom.xml changes
+RUN mvn -q -ff -T 1C install && rm -rf target
+
+COPY management-api-agent ./management-api-agent
+COPY management-api-common ./management-api-common
+COPY management-api-server ./management-api-server
+COPY management-api-shim-3.x ./management-api-shim-3.x
+COPY management-api-shim-4.x ./management-api-shim-4.x
+RUN mvn -q -ff package -DskipTests
+
+FROM --platform=$BUILDPLATFORM maven:3.6.3-jdk-8-slim as netty4150
+RUN mvn dependency:get -DgroupId=io.netty -DartifactId=netty-all -Dversion=4.1.50.Final -Dtransitive=false
+
+FROM --platform=linux/amd64 cassandra:3.11 as oss311-amd64
+
+FROM --platform=linux/arm64 cassandra:3.11 as oss311-arm64
+# Netty arm64 epoll support was not added until 4.1.50 (https://github.com/netty/netty/pull/9804)
+# Only replace this dependency for arm64 to avoid regressions
+RUN rm /opt/cassandra/lib/netty-all-*.jar
+COPY --from=netty4150 /root/.m2/repository/io/netty/netty-all/4.1.50.Final/netty-all-4.1.50.Final.jar /opt/cassandra/lib/netty-all-4.1.50.Final.jar
```

Comment:
The better way to do this is probably to have the management api agent contain a shaded version of netty at the appropriate version. That will take further experimentation. However, we are only swapping out this jar for arm64, so this should not impact any existing users.